src/main/java/DataManipulation.java

```diff
@@ -0,0 +1,76 @@
+import static java.util.Arrays.asList;
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.List;
+import org.apache.arrow.memory.RootAllocator;
+import org.apache.arrow.vector.FieldVector;
+import org.apache.arrow.vector.VarCharVector;
+import org.apache.arrow.vector.VectorSchemaRoot;
+
+
+public class DataManipulation {
+  public static final RootAllocator allocator = new RootAllocator(Long.MAX_VALUE);
+
+  final VarCharVector nameVector = new VarCharVector("name", allocator);
+  final VarCharVector teamVector = new VarCharVector("team", allocator);
+  final VarCharVector positionVector = new VarCharVector("position",allocator);
+  final VarCharVector heightVector = new VarCharVector("height", allocator);
+  final VarCharVector weightVector = new VarCharVector("weight", allocator);
+  final VarCharVector  ageVector = new VarCharVector("age", allocator);
+
+  public void readCSVFile(String path) throws FileNotFoundException {
+    String line;
+    try {
+      BufferedReader br = new BufferedReader(new FileReader(path));
```

Comment:
Add this to a try with resources, so you don't need to worrying closing it

Reply:
Resolved.